lib/marqeta/one_time.rb

```diff
@@ -0,0 +1,7 @@
+module Marqeta
+  class OneTime < ApiObject
```

Comment:
@sughandj Curious. Why is the class name "OneTime" ?

Reply:
@teresa-lo Marqeta gives a "OneTime" object at `users/auth/onetime` endpoint which has a token field. We do `onetime.token` to get the token.